diff --git a/src/items/powerup.cpp b/src/items/powerup.cpp
--- a/src/items/powerup.cpp
+++ b/src/items/powerup.cpp
@@ -62,7 +62,7 @@ Powerup::~Powerup()
  */
 void Powerup::reset()
 {
-    m_type = PowerupManager::POWERUP_NOTHING;
+    m_type = PowerupManager::POWERUP_ZIPPER;//POWERUP_NOTHING;
     m_number = 0;

     // Ghost kart will update powerup every frame
@@ -495,7 +495,7 @@ void Powerup::use()
     if ( m_number <= 0 )
     {
         m_number = 0;
-        m_type   = PowerupManager::POWERUP_NOTHING;
+        m_type   = PowerupManager::POWERUP_ZIPPER;//POWERUP_NOTHING;
     }
 }   // use

diff --git a/src/karts/kart.cpp b/src/karts/kart.cpp
--- a/src/karts/kart.cpp
+++ b/src/karts/kart.cpp
@@ -373,7 +373,7 @@ void Kart::reset()
     m_invulnerable_ticks   = 0;
     m_min_nitro_ticks      = 0;
     m_energy_to_min_ratio  = 0;
-    m_collected_energy     = 0;
+    m_collected_energy     = 1;
     m_bounce_back_ticks    = 0;
     m_brake_ticks          = 0;
     m_ticks_last_crash     = 0;
@@ -1597,11 +1597,11 @@ void Kart::update(int ticks)
         const float roll = quad_normal.angle
                ((Vec3(0, 1, 0).rotate(q.getAxis(), q.getAngle())));

-        if (Track::getCurrentTrack()->isAutoRescueEnabled() &&
+        if (/*Track::getCurrentTrack()->isAutoRescueEnabled() &&
             (!m_terrain_info->getMaterial() ||
             !m_terrain_info->getMaterial()->hasGravity()) &&
             !has_animation_before && fabs(roll) > 60 * DEGREE_TO_RAD &&
-            fabs(getSpeed()) < 3.0f)
+            fabs(getSpeed()) < 3.0f*/false)
         {
             RescueAnimation::create(this, /*is_auto_rescue*/true);
             m_last_factor_engine_sound = 0.0f;
@@ -1691,8 +1691,8 @@ void Kart::update(int ticks)
         const Vec3 *min, *max;
         Track::getCurrentTrack()->getAABB(&min, &max);

-        if((min->getY() - getXYZ().getY() > 17 || dist_to_sector > 25) && !m_flying &&
-           !has_animation_before)
+        if(/*(min->getY() - getXYZ().getY() > 17 || dist_to_sector > 25) && !m_flying &&
+           !has_animation_before*/false)
         {
             RescueAnimation::create(this);
             m_last_factor_engine_sound = 0.0f;
@@ -1700,7 +1700,7 @@ void Kart::update(int ticks)
     }
     else
     {
-        if (!has_animation_before && material->isDriveReset() && isOnGround())
+        if (false && !has_animation_before && material->isDriveReset() && isOnGround())
         {
             RescueAnimation::create(this);
             m_last_factor_engine_sound = 0.0f;
@@ -2195,7 +2195,7 @@ void Kart::handleZipper(const Material *material, bool play_sound)
         if(duration<0)
             duration           = m_kart_properties->getZipperDuration();
         if(speed_gain<0)
-            speed_gain         = m_kart_properties->getZipperSpeedGain();
+            speed_gain         = m_kart_properties->getZipperSpeedGain()*16;
         if(fade_out_time<0)
             fade_out_time      = m_kart_properties->getZipperFadeOutTime();
         if(engine_force<0)
@@ -2433,7 +2433,7 @@ void Kart::crashed(const Material *m, const Vec3 &normal)
             }
         }
 #endif
-        if (m->getCollisionReaction() == Material::RESCUE)
+        if (false && m->getCollisionReaction() == Material::RESCUE)
         {
             RescueAnimation::create(this);
             m_last_factor_engine_sound = 0.0f;
diff --git a/src/karts/kart_properties.cpp b/src/karts/kart_properties.cpp
--- a/src/karts/kart_properties.cpp
+++ b/src/karts/kart_properties.cpp
@@ -1150,19 +1150,19 @@ float KartProperties::getNitroDuration() const
 // ----------------------------------------------------------------------------
 float KartProperties::getNitroEngineForce() const
 {
-    return m_cached_characteristic->getNitroEngineForce();
+    return m_cached_characteristic->getNitroEngineForce()*2;
 }  // getNitroEngineForce

 // ----------------------------------------------------------------------------
 float KartProperties::getNitroEngineMult() const
 {
-    return m_cached_characteristic->getNitroEngineMult();
+    return m_cached_characteristic->getNitroEngineMult()*2;
 }  // getNitroEngineMult

 // ----------------------------------------------------------------------------
 float KartProperties::getNitroConsumption() const
 {
-    return m_cached_characteristic->getNitroConsumption();
+    return m_cached_characteristic->getNitroConsumption()*0;
 }  // getNitroConsumption

 // ----------------------------------------------------------------------------
@@ -1192,7 +1192,7 @@ float KartProperties::getNitroFadeOutTime() const
 // ----------------------------------------------------------------------------
 float KartProperties::getNitroMax() const
 {
-    return m_cached_characteristic->getNitroMax();
+    return m_cached_characteristic->getNitroMax()*64;
 }  // getNitroMax

 // ----------------------------------------------------------------------------
diff --git a/src/karts/max_speed.cpp b/src/karts/max_speed.cpp
--- a/src/karts/max_speed.cpp
+++ b/src/karts/max_speed.cpp
@@ -55,7 +55,7 @@ MaxSpeed::MaxSpeed(AbstractKart *kart)
     // update() is called.
     m_add_engine_force  = 0;
     // This can be used if command line option -N is used
-    m_current_max_speed = 0;
+    m_current_max_speed = 340282346638528859811704183484516925440.0f;//0;
 }   // MaxSpeed

 // ----------------------------------------------------------------------------
@@ -65,7 +65,7 @@ MaxSpeed::MaxSpeed(AbstractKart *kart)
  */
 void MaxSpeed::reset()
 {
-    m_current_max_speed = m_kart->getKartProperties()->getEngineMaxSpeed();
+    m_current_max_speed = 340282346638528859811704183484516925440.0f;//m_kart->getKartProperties()->getEngineMaxSpeed();
     m_min_speed         = -1.0f;

     for(unsigned int i=MS_DECREASE_MIN; i<MS_DECREASE_MAX; i++)
@@ -414,7 +414,7 @@ void MaxSpeed::update(int ticks)
     }

     m_add_engine_force  = 0;
-    m_current_max_speed = m_kart->getKartProperties()->getEngineMaxSpeed();
+    m_current_max_speed = 340282346638528859811704183484516925440.0f;//_kart->getKartProperties()->getEngineMaxSpeed();

     // Then add the speed increase from each category
     // ----------------------------------------------
diff --git a/src/tracks/track.hpp b/src/tracks/track.hpp
--- a/src/tracks/track.hpp
+++ b/src/tracks/track.hpp
@@ -666,7 +666,7 @@ public:
     bool isInternal() const { return m_internal; }
     // ------------------------------------------------------------------------
     /** Returns true if auto rescue is enabled. */
-    bool isAutoRescueEnabled() const { return m_enable_auto_rescue; }
+    bool isAutoRescueEnabled() const { return false/*m_enable_auto_rescue*/; }
     // ------------------------------------------------------------------------
     /** True if push back of karts towards the track should be enabled. */
     bool isPushBackEnabled() const { return m_enable_push_back; }
